src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Deposit/components/Confirm.tsx

```diff
@@ -206,6 +207,54 @@ export const Confirm: React.FC<ConfirmProps> = ({ accountId, onNext }) => {
     fromAddress: fromAddress ?? null,
   })
 
+  const { amountCryptoBaseUnitMinusFees, amountCryptoHumanMinusFees } = useMemo(() => {
+    if (!estimatedFeesData)
```

Comment:
For UTXO chains, we should early return with the values from state, which are already fees-deducted

Reply:
Fixed in [785cec8](https://github.com/shapeshift/web/pull/7606/commits/785cec859251e10c00102f88be7c1bab7d28963f)